.travis.yml

```diff
@@ -6,8 +6,10 @@ language: node_js
 sudo: false
 
 before_install:
+  - pip install restructuredtext_lint
```

Comment:
Sorry if I missed this issue in https://github.com/openmicroscopy/omero-weberror/pull/11/ ?